manager/state/raft/raft.go

```diff
@@ -1647,7 +1652,7 @@ func getIDs(snap *raftpb.Snapshot, ents []raftpb.Entry) []uint64 {
 		case raftpb.ConfChangeUpdateNode:
 			// do nothing
 		default:
-			log.G(context.Background()).Panic("ConfChange Type should be either ConfChangeAddNode or ConfChangeRemoveNode!")
+			log.G(ctx).Panic("ConfChange Type should be either ConfChangeAddNode or ConfChangeRemoveNode!")
```

Comment:
Just noticed that this log message is misleading. `ConfChangeUpdateNode` is also valid.